build_tools/cmake/build_runtime_tool_and_test_benchmark_suites.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Build tests based on benchmark suites for IREE.
+# Designed for CI, but can be run locally.
```

Comment:
`build_tools/cmake/build_runtime_tool_and_test_benchmark_suites.sh` is a pretty long name, doing something pretty specific... can we just extend one of the existing scripts instead? What is making that difficult/awkward?

Reply:
I was hoping we could build less target (`iree-run-module` and test targets) instead of the full set. I could reuse the existing flow, but it could build more targets than needed. I will update this flow.

Reply:
Doesn't build_riscv already only build the runtime? I think at the point you're building `iree-run-module` you're already building the entire runtime, so not sure there's much difference

Reply:
This is used by x86_64 benchmark suite module test. The challenge is that https://github.com/iree-org/iree/blob/7cb39b0/build_tools/cmake/iree_native_test.cmake#L126 looks for a target for `iree-run-module`, and we have to build the target since it is defined in the host build (use `IREE_HOST_BINARY_ROOT` and `iree_get_executable` on host build will cause a target collision between the aliased one and the actual one). `iree_run_module` target needs to be defined during the test configuration time otherwise the cmake configuration will fail.

If there is a better way to avoid this collision, I am happy to apply. Please advise.

Reply:
I updated the script name to `test_benchmark_suite_host.sh` to indicate what needs to be done to run the benchmark suite test in a host setup. Happy to take other advise if there's a better approach.

Reply:
Move the discussion to https://github.com/iree-org/iree/pull/10652